src/client/ua_client.c

```diff
@@ -1,6 +1,11 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ *
+ *    Copyright 2015-2017 (C) Julius Pfrommer
```

Comment:
Please add #430
Copyright 2015 (C) Holger Jeromin

Reply:
Looks like refactoring of the code kicked you out of the git blame. You should definitely be in the copyright holders list! Sorry for that.

@open62541/open62541-core 
Hmm, should we then use the whole history of a file and sum up all the contributions. If more than 10 lines have been changed the author is added, instead of only using the current status/git blame?